PowerShell Scanners/SQL Server Databases/README.md

```diff
@@ -0,0 +1,17 @@
+# Instructions
+[How to use this repository](../../README.md)
+
+# Description
+Returns a list of databases for a given SQL Server instance.
+The information includes id, name, compatibility level, creation date, owner, collation and status.
```

Comment:
Markdown has special requirements for a newline: either 2 spaces at the end of the line, or an empty line between lines. As is, all 3 of these sentences are in 1 paragraph. It looks fine, but it may not be what you were expecting.

Reply:
Other than that, looks good to me. I'll give it a few days for any changes to the markdown and we can get it merged after that. Thanks, all!